plugins/CoreHome/templates/piwikIsOutdated.twig

```diff
@@ -0,0 +1,60 @@
+
+            </p>
+
+            <h2>How to upgrade</h2>
+            <p>The update to Matomo (Piwik) 3 should be smooth, but may take a while depending on the amount of data you have. If you have any problem with the update, feel free to  <a href="https://matomo.org/support">get in touch</a> with us. At Matomo and at <a href="https://www.innocraft.com" onclick="javascript:window.open('https://www.innocraft.com/'); return false;">InnoCraft</a>, the company of the makers of Matomo, we have successfully updated many Matomo installations.</p>
+            <p>    If your server meets the requirements for Matomo 3, you can <a href="https://matomo.org/docs/update/">upgrade your Piwik with our automatic updater</a>.
+                If not, you may need to upgrade your server before you can update to the latest server version.
+            </p>
+
+            <h2>Don't want the hassle of updating and maintaining your Piwik? Migrate to the Matomo Analytics Cloud</h2>
+            <p>
+                We can migrate your Piwik database to our Matomo Analytics Cloud without the loss of any data while you keep 100% data ownership.<br />
+                The Cloud is operated directly by the makers of Piwik/Matomo and is reliable, secure, comes with bonus features, and always kept up to date.
+                <br /><br />
+                <a href="https://www.innocraft.cloud/">Learn more about the Matomo Analytics Cloud</a><br />
+                <a href="https://www.innocraft.cloud/#pricing">View Pricing</a><br />
+                <a href="https://www.innocraft.cloud/#contact">Get in touch with our support team to learn more about the migration</a><br />
+            </p>
+
+            {% if isSuperUser %}
+            <h2>Want to disable this warning? (not recommended)</h2>
+            <p>
+                A user with access to the server can edit the <code>config/config.ini.php</code> file and add the following lines (the line with <code>[General]</code> may already exist):
+                <br />
+                <br />
+                <pre><code>[General]
+show_piwik2_end_of_life_warning = 0</code></pre>
+            </p>
+            {% endif %}
+        </div>
+    </div>
+{% endblock %}
```

Comment:
I guess this shouldn't be translated?

Reply:
No translation needed.  Discussed it in the chat a while ago. It wouldn't be translated in time likely anyway and it'll be the last update.